cosmos/dbt/graph.py

```diff
@@ -196,8 +203,8 @@ def load_via_custom_parser(self):
             node = DbtNode(
                 name=model_name,
                 unique_id=model_name,
-                resource_type=model.type,
-                depends_on=model.config.upstream_models,
+                resource_type=model.type,  # type: ignore[arg-type] # DbtModelType is not a str
```

Comment:
I'm fixing this in #375 - if we merge #375 before, we should remove this ignore clause.

Reply:
Resolving in a separate, future ticket.